src/pcapi/core/bookings/repository.py

```diff
@@ -303,6 +300,10 @@ def _filter_bookings_recap_query(
         .join(UserOfferer)
         .filter(UserOfferer.userId == user_id)
         .filter(UserOfferer.validationToken.is_(None))
+        .filter(
+            booking_date <= booking_period_ending_date,
+            booking_date >= booking_period_beginning_date,
```

Comment:
On devrait pouvoir utiliser le mot-clé SQL [BETWEEN](https://www.postgresql.org/docs/12/functions-comparison.html) et ainsi éviter de "décomposer" ("déstructurer" ?) `booking_period` en 2 variables temporaires :

```suggestion
            booking_date.between(*booking_period, symmetric=True),
```

`symmetric=True` permet d'indiquer à PostgreSQL d'inverser automatiquement les 2 bornes si on ne sait pas quelle est la borne inférieure et la borne supérieure.

Reply:
🤩